src/util/DefaultValueProcessor.ts

```diff
@@ -0,0 +1,41 @@
+export const addValues = (testCase: any): any => {
+  //create a clone of testCase
+  const resultJson: any = testCase;
+  //.map to return an array of just
+  const nonCoverage: Array<any> = testCase.entry.filter((entry) => {
+    if (entry.resource.resourceType != "Coverage") {
+      return true;
+    }
+  });
+
+  const coverage: Array<any> = [];
+  const foundCoverage: any = testCase.entry.find((entry) => {
+    if (entry.resource.resourceType == "Coverage") {
+      entry.resource.status = "active";
```

Comment:
This mutates the entry resource from the input parameter

Reply:
Is this resolved by using deepClone and then using the cloned object instead of the input parameter?

Reply:
yes it would fix that issue